setup.py

```diff
@@ -50,6 +51,9 @@
         'openpyxl',
         'packaging',
         'pandas',
+        'pyproj>=2.1',
+        # 'pys2index',  # issues installing from PyPI (wheel doesn't build)
+        'pyyaml',
```

Comment:
Please use alphabetical ordering

Reply:
it is, isn't it though?

Reply:
It seems `pyproj` and `pyyaml` appears twice in `setup.py`. They were added by the first commit in this PR, and they've been added in the main branch as well. Probably merging error, I'll fix it.

Reply:
ah completely missed that - am actually gonna do a technical review of this one (might as well, since I poked around it anyway) :+1: